src/diversity/index.tsx

```diff
@@ -0,0 +1,163 @@
+import { ButtonLink } from '@components/button';
+import Head from 'next/head';
+import SayHi from 'src/index/say-hi';
+import Layout from 'src/layout';
+import style from './diversity.module.css';
+
+const Diversity = () => {
+  return (
+    <Layout title="Variant – Mangfold">
+      <Head>
+        <meta
+          property="og:description"
+          content="Vi i Variant samarbeider med de som aktivt jobber for å skape mangfold, fordi det gir verdi og nytte for absolutt alle."
+        />
+        <meta
+          name="description"
+          content="Vi i Variant samarbeider med de som aktivt jobber for å skape mangfold, fordi det gir verdi og nytte for absolutt alle.."
+        />
+      </Head>
+      <SayHi
+        className={style.sayHi}
+        href="https://twitter.com/intent/tweet?screen_name=variant_as"
+        rel="noopener"
+      />
+
+      <header className={style.header}>
+        <h2 className={style.header__title}>Mangfold</h2>
```

Comment:
burde ikke dette være en h1? 🤔

Reply:
Joooo, bør kanskje det. Har egentlig bare kopiert koden for tittelen fra forsida på variant.no (som er h2...)

Kan endre på det jeg

Reply:
h1 er satt opp til logo/variant, h2 side-innhold. Følger samme informasjonshierarki som resten av variant.no

Reply:
Bedre å rydde opp fra nå av enn å merge inn flere "feil" tenker jeg da :D

Reply:
Det er ikke "feil" IMO. Det er vanlig praksis at Nettstedsnavn er h1, sideinnhold er h2 osv. Etter min mening er dette tydelig bestepraksis for informasjonshierarki

Reply:
> 

Blir det riktig bruk av headings da?

Reply:
Tenkte mest på bruk av headings for best mulig accessibility. Ikke kjempenyttig at skjermleseren leser opp Variant hver gang en går inn på en ny side eller? Men skal ikke krangle. Bare stilte spørsmål ved det

Reply:
Det er nettopp pga accessibility og skjermlesere at det er en vanlig praksis mener jeg. F.eks ved navigerer av document outline er det praktisk å kunne hoppe til toppen av siden (h1) for å traversere lengre ned på menyer osv. SEO-messig kan det og være en fordel da det signaliserer rank av titler. Det kan også være at du går i mellom forskjellige tabs og det er nyttig at det er synliggjort hvilken side du er på. Nå har du også dette via title, da. Nå er det riktignok [noen år](http://meyerweb.com/eric/thoughts/2004/07/21/pick-a-heading/) siden dette har vært oppe til diskusjon, så jeg er åpen for at bestepraksis eller praksis på dette er byttet. 😅

Reply:
> Blir det riktig bruk av headings da?

Ja, det blir i alle fall ikke _uriktig_ bruk. Eksempel: 
- https://www.w3.org/WAI/tutorials/page-structure/headings/
- https://fae.disability.illinois.edu/rulesets/HEADING_1/

Når det er sagt er det jo alt etter bruksmønster til de med ulike skjermlesere. Ja, SEO, er en del av det, men viktigste er jo skjermlesere her. Så hadde vært interessant å snakket med noen som bruker skjermlesere hver dag til å høre hva de tenker. Dette er en diskusjon jeg har hatt mange ganger, og per nå er konklusjonen ofte "det finnes ingen fasit". Så vært fint å finne en fasit 😄

Reply:
Mulig vi kunne vært litt striktere og sagt at *kun variant.no/* (uten underside) har Variant som h1, men på andre sider er det innhold-tittel som er h1 🤔

Reply:
Merker at denne diskusjonen fort ble litt for filosofisk til scopet på denne PRen, så merget inn nå jeg ;)

Reply:
Vi kan jo diskutere mer / ta en skjermleser-sesjon på Variantdagen i oktober @mikaelbr ? 😊 bra du setter grenser, @itzjacki 👏🏼